src/Integration.Vsix.UnitTests/CFamily/VcxRequestFactoryTests.cs

```diff
@@ -19,19 +19,27 @@
  */
 
 using System;
+using System.Collections.Generic;
 using EnvDTE;
 using FluentAssertions;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using SonarLint.VisualStudio.CFamily.Analysis;
+using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.CFamily;
 using SonarLint.VisualStudio.Integration.Vsix.CFamily;
+using SonarLint.VisualStudio.Integration.Vsix.CFamily.VcxProject;
+using static SonarLint.VisualStudio.Integration.Vsix.CFamily.UnitTests.CFamilyTestUtility;
 
 namespace SonarLint.VisualStudio.Integration.UnitTests.CFamily
 {
     [TestClass]
     public class VcxRequestFactoryTests
```

Comment:
I copied some of the tests from CFamilyHelperTests, including the method GetSuccessfulRequest.
However, I had to modify things so I think it's better to treat all of them as "new".